app/src/CardFlipGame.tsx

```diff
@@ -1,22 +1,42 @@
 import styled from "styled-components";
 import BackButton from "./BackButton";
 import NextButton from "./NextButton";
+import { ErrorBoundary } from "react-error-boundary";
+import ErrorFallback from "./App";
 
-interface Props {}
+export default function CardFlipGame(): JSX.Element {
```

Comment:
I updated `CardFlipGame`  function to a JSX.Element as it was causing the error in `App` "JSX element type 'CardFlipGame' does not have any construct or call signatures" I think that typescript can more easily infer types by using this, and it won't complain in VSCode with the dreaded red lines